modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/notification-sidebar/index.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import classNames from 'classnames';
+import {useEffect, useState} from 'react';
+
+import './index.scss';
+import {
+	getUserNotification,
+	putUserNotificationRead,
+} from '../../../common/services/Notification';
```

Comment:
```suggestion
} from '../../../common/services/notification';
```
Since the notification file isn't a component, the ideal is that we should not use pascal case only in cases of .jsx/.tsx components. Is a convention to make the code more readable

Reply:
thanks for explaining.